docs/usage.md

```diff
@@ -1,45 +1,8 @@
-# qbic-pipelines/bamtofastq: Usage
-
-## Table of contents
-
-<!-- Install Atom plugin markdown-toc-auto for this ToC to auto-update on save -->
```

Comment:
since we can't render the docs on the website, a TOC might be nice to keep (no fancy side bar etc on the github markdown rendering)